src/main/java/equipment/describer/TeamEquipmentDescriber.java

```diff
@@ -0,0 +1,22 @@
+package equipment.describer;
+
+import equipment.AbstractEquipment;
+import equipment.describer.EquipmentDescriber;
+import team.ITeamMember;
+
+import java.util.List;
+import java.util.Map;
+
+public class TeamEquipmentDescriber {
+    public String getTeamEquipmentDescription(Map<ITeamMember, List<AbstractEquipment>> teamEquipments) {
```

Comment:
if you only use the keys of the map, why not just get a list of team members and use their `getEquipment` method?